drivers/wifi/nrf_wifi/inc/fmac_main.h

```diff
@@ -88,6 +88,8 @@ struct nrf_wifi_vif_ctx_zep {
 	struct k_work_delayable nrf_wifi_rpu_recovery_bringup_work;
 #endif /* CONFIG_NRF_WIFI_RPU_RECOVERY */
 	int rts_threshold_value;
+	bool bss_max_idle_period_user_set;
```

Comment:
as this is just a once shot flag, do we really need it? We can just initialized `bss_max_idle_period = -1` (which overflows as it's unsigned int and becomes invalid as valid is only 64000`?